include/kumi/tuple.hpp

```diff
@@ -39,7 +40,40 @@ namespace kumi
     static constexpr bool is_homogeneous = binder_t::is_homogeneous;
 
     binder_t impl;
+    
+    /// Static helper to get access to the underlying value of a kumi::member_capture
+    template<typename U>
```

Comment:
Those things should not live in tuple but outside; they'll get instantiated for every different tuple type for nothing.